sdk/communication/azure-communication-common/pom.xml

```diff
@@ -46,12 +46,12 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.30.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
```

Comment:
azure-communication-common
is this something we should change in this PR?

Reply:
Yes, there is some changes in contentDownloader part, basically some features are refactored into new version's Azure Core. This change was required from the reviewer.